composer.json

```diff
@@ -156,8 +157,7 @@
         "twig/cssinliner-extra": "^2.12|^3",
         "twig/inky-extra": "^2.12|^3",
         "twig/markdown-extra": "^2.12|^3",
-        "web-token/jwt-checker": "^3.1",
-        "web-token/jwt-signature-algorithm-ecdsa": "^3.1"
+        "web-token/jwt-library": "3.3.x-dev"
```

Comment:
please use `^3.3` instead, which will avoid having to change the requirement once tagged (our dev setup runs with `minimum-stability: dev` anyway)

Reply:
`web-token/jwt-library` is a new library which will tagged in few days.
https://github.com/symfony/symfony/pull/53682#issuecomment-1922051301